app/config/services.yml

```diff
@@ -4,6 +4,5 @@ parameters:
 #    parameter_name: value
 
 services:
-#    service_name:
-#        class: AppBundle\Directory\ClassName
+#    'My\ClassName':
```

Comment:
what both syntaxes ? We don't have 2 syntaxes. We just made ``class`` default to the id when it is omitted. The id of your service is just ``My\ClassName`` in this case.
You are free to keep using other ids if you prefer btw. Services will not be forced to use a class name as id.